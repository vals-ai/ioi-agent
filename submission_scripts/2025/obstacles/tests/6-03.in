version https://git-lfs.github.com/spec/v1
oid sha256:91930832730ae8c054c73bf14ea8d9d9d7d3e36b4b5234f1e5daa45d39eca9d7
size 7906639
