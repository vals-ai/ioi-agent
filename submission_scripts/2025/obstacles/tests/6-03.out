version https://git-lfs.github.com/spec/v1
oid sha256:71f464da6f6790983af553d4f50ffc184c45432dd02382347e39b92def174061
size 400036
